cedar-policy/CHANGELOG.md

```diff
@@ -52,6 +52,10 @@ Cedar Language Version: 4.0
 - (*) Implemented [RFC 52](https://github.com/cedar-policy/rfcs/blob/main/text/0052-reserved-namespaces.md).
   Names containing `__cedar` (e.g., `__cedar`, `A::__cedar`, `__cedar::A`, and
 `A::__cedar::B`) are now invalid. (#969)
+- Replace uses of "natural", "human", "human-readable", and "custom" with "Cedar" (#1114).
+  APIs with these names are changed accordingly. E.g., `Schema::from_str_natural` to `Schema::from_cedarschema_str`.
+  Moreover, the `FromStr` implementations of `Schema` and `SchemaFragment`
+  now parse strings in the Cedar schema format.
```

Comment:
nit: maybe add that you can still parse the JSON schema format using [method name]

Reply:
Done.